README.md

```diff
@@ -4,32 +4,41 @@
 [![Actions Status: build](https://github.com/idan-at/eslint-plugin-package-json-dependencies/workflows/test/badge.svg)](https://github.com/idan-at/eslint-plugin-package-json-dependencies/actions?query=workflow%3A"test")
 
 
-This plugin contains rules for valid and consistent package json dependencies.
+This plugin contains rules for maintaining a valid, consistent `package.json` dependency setup.
 
-# Install
-Make sure `eslint` is installed.
+# Installation
 
-Then,
-
-`npm install --save-dev eslint-plugin-package-json-dependencies`
+1. Make sure `eslint` is installed, by runnning:
+```bash
+eslint --version
+```
+If you get an error message, run:
+```bash
+npm install -g eslint
```

Comment:
I think we better suggest installing it locally:
```suggestion
npm install --save-dev eslint
```

and I think we can just combine everything into a single installation step:
```bash
npm install --save-dev eslint eslint-plugin-package-json-dependencies
```